db/migrate/20210823144420_add_updated_status_columns_to_sent_hearing_email_events.rb

```diff
@@ -0,0 +1,6 @@
+class AddUpdatedStatusColumnsToSentHearingEmailEvents < Caseflow::Migration
+  def change
+    add_column :sent_hearing_email_events, :email_sent, :boolean, comment: "whether the email was sent or not."
+    add_column :sent_hearing_email_events, :sent_status_checked_at, :datetime, comment: "The date the status was last checked/updated in the GovDelivery API."
```

Comment:
Really minor but for consistency with other comments:
```suggestion
    add_column :sent_hearing_email_events, :email_sent, :boolean, comment: "whether the email was sent or not"
    add_column :sent_hearing_email_events, :sent_status_checked_at, :datetime, comment: "The date the status was last checked/updated in the GovDelivery API"
```